tools/roslaunch/src/roslaunch/remoteprocess.py

```diff
@@ -203,7 +203,7 @@ def _ssh_exec(self, command, address, port, username=None, password=None):
                 err_msg = "Unable to establish ssh connection to [%s%s:%s]: %s"%(username_str, address, port, e)
             except socket.error as e:
                 # #1824
-                if e[0] == 111:
+                if e.args[0] == 111:
```

Comment:
How is this change related to the issue title?

Reply:
I updated the description. Also, changing `e[0]` to `e.args[0]` wasn't really related - I will create separate PRs the next time. Given that I have deleted my fork, is there any way to fix this on my end? Alternatively, you could cherry pick the first commit and ignore the second.

Reply:
Thank you for the additional information.

With the forked repo being delete I am not sure I can cherry-pick individual commits. I will just squash both commits during the merge in this case.